.github/workflows/k8s_e2e.yml

```diff
@@ -18,10 +21,12 @@ on:
     types: [created]
   merge_group:
     types: [checks_requested]
+  schedule:
+    # At midnight UTC Tue-Sat
+    - cron: '0 0 * * 2-6'
```

Comment:
How will we be notified if the scheduled build fails? I just want to make sure it doesn't slip through the cracks. Maybe we could take this opportunity to setup a Slack notification?

Reply:
👀 Datadog Monitor 👀

Reply:
Doh, yeah that's kinda important lol.
I'll take a look at adding a Datadog Monitor.

Reply:
In addition to looking at Datadog Monitors, I'd also evaluate https://github.blog/changelog/2022-12-06-github-actions-workflow-notifications-in-slack-and-microsoft-teams/ .

Reply:
After some research and discussion, we will try the GHA workflow notifications in slack first and pivot if needed.
There aren't any edits needed to the workflow file itself for that.